pallets/oracle/Cargo.toml

```diff
@@ -20,6 +20,9 @@ frame-system = { git = "https://github.com/paritytech/substrate", branch = "polk
 frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", default-features = false, optional = true }
 pallet-timestamp = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", default-features = false }
 
+orml-oracle = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v0.9.33", default-features = false }
```

Comment:
We updated orml dependencies to `0.9.36` on the dev/milestone-3 branch yesterday. So you should merge the dev branch into this one and change the branch here to `polkadot-v0.9.36`.

Reply:
Thanks

Reply:
Done